src/pages/Collections/Collections.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import * as S from './Collections.style';
+
+export default function Collections() {
+  const [houseDataList, setHouseDataList] = useState([]);
+
+  //TOFIX: mockData 연결 시 동작할 코드
+  useEffect(() => {
+    fetch('./data/houseWarming.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setHouseDataList(data.data);
+      });
+  }, []);
+
+  return (
+    <S.Main>
+      <S.CollectionsMain>
+        <div>
+          <S.Title>스크랩 북</S.Title>
+          <S.Profile>
+            <S.ProfileImg
+              src="https://cdn-icons-png.flaticon.com/512/3746/3746927.png"
+              alt="profileImg"
+            />
+            <S.NicKName>hahahoho</S.NicKName>
+          </S.Profile>
+        </div>
+        <S.HouseContentsBox>
+          <S.ContentTitle>집들이({houseDataList.length})</S.ContentTitle>
+          <S.ContentsBox>
+            {houseDataList.length === 0 ? (
+              <>
+                <S.HeadContent>아직 스크랩한 콘텐츠가 없습니다.</S.HeadContent>
+                <S.Content>
+                  마음에 드는 콘텐츠를 발견하면 스크랩해보세요
+                </S.Content>
+              </>
+            ) : (
+              <S.HouseWarmingBox>
+                {houseDataList.map(houseWarmingList => {
+                  return (
+                    <S.HouseWarmingLink to="/detail" key={houseWarmingList.id}>
+                      <S.HouseWarmimgImg
+                        src={houseWarmingList.imageUrl}
```

Comment:
```suggestion
                {houseDataList.map(({id,imageUrl})=> {
                  return (
                    <S.HouseWarmingLink to="/detail" key={id}>
                      <S.HouseWarmimgImg
                        src={imageUrl}
```

접근하고 싶은건 key이고, key가 많지않아 구조분해할당 해주면 바로 접근할 수 있습니다.